commit 54e714fafc7e56f717cda0c418273fdc8350f3ef (refs/changes/43/3743/3)
Author: Yuka Poppe <yuka@netbox.org>
Date:   2018-04-19T23:24:01+00:00 (1 year, 6 months ago)
    
    TRITON-289 "booter menu-lst" and "booter boot-ipxe" commands don't work

diff --git a/lib/booter.js b/lib/booter.js
index 5fa83eb..f278a21 100644
--- a/lib/booter.js
+++ b/lib/booter.js
@@ -73,6 +73,7 @@ function main() {
     case 'menu-lst':
         bootparams.getBootParams({
             adminUuid: config.adminUuid,
+            tftpRoot: config.tftpRoot,
             cacheDir: config.cache.dir,
             mac: getArg('MAC address'),
             napi: napi,
@@ -83,7 +84,7 @@ function main() {
                 return console.error(err.code + ': ' + err.message);
             }
 
-            return menu.buildMenuLst(res, config.tftpRoot, function (lst) {
+            return menu.buildMenuLst(res, function (lst) {
                 return console.log(lst);
             });
         });
@@ -94,6 +95,7 @@ function main() {
     case 'boot-ipxe':
         bootparams.getBootParams({
             adminUuid: config.adminUuid,
+            tftpRoot: config.tftpRoot,
             cacheDir: config.cache.dir,
             mac: getArg('MAC address'),
             napi: napi,
@@ -104,7 +106,7 @@ function main() {
                 return console.error(err.code + ': ' + err.message);
             }
 
-            menu.buildIpxeCfg(res, config.tftpRoot, function (cfg) {
+            menu.buildIpxeCfg(res, function (cfg) {
                 return console.log(cfg);
             });
         });
